Argo/Types/StandardTypes.swift

```diff
@@ -72,11 +157,46 @@ public extension CollectionType where Generator.Element: Decodable, Generator.El
   }
 }
 
+/**
+  Decode `JSON` into an array of values where the elements of the array are
+  `Decodable`.
+
+  If the `JSON` is an array of `JSON` objects, this returns a decoded array
+  of values by mapping the element's `decode` function over the `JSON` and
+  then `sequencing` the result. This makes this `Decode` function an
+  all-or-nothing operation (See the documentation for `sequence` for more info).
+
+  If the `JSON` is not an array, this returns a type mismatch.
+
+  This is a convenience function that is the same as `[T].decode(j)` (where `T`
+  is `Decodable`) and only exists to ease some pain around needing to use the
+  full type of the array when calling `decode`. We expect this function to be
+  removed in a future version.
+
+  - parameter j: The JSON calue to decode
+
+  - returns: A decoded array of values
+*/
 public func decodeArray<T: Decodable where T.DecodedType == T>(j: JSON) -> Decoded<[T]> {
   return [T].decode(j)
 }
 
 public extension DictionaryLiteralConvertible where Value: Decodable, Value == Value.DecodedType {
+  /**
+    Decode `JSON` into a dictionary of keys and values where the keys are
+    `String`s and the values are `Decodable`.
+
+    If the `JSON` is a dictionary of `String`/`JSON` pairs, this returns a decoded dictionary
+    of key/value pairs by mapping the value's `decode` function over the `JSON` and
```

Comment:
the slash makes more sense with key/value ... so I guess it's fine above if you want to keep consistency